Tools/cases_generator/generate_cases.py

```diff
@@ -604,6 +631,38 @@ def write_executor_instructions(
             file=sys.stderr,
         )
 
+    def write_abstract_interpreter_instructions(
+        self, abstract_interpreter_filename: str, emit_line_directives: bool
+    ) -> None:
+        """Generate cases for the Tier 2 abstract interpreter/analzyer."""
+        with open(abstract_interpreter_filename, "w") as f:
+            self.out = Formatter(f, 8, emit_line_directives)
+            self.write_provenance_header()
+            for thing in self.everything:
+                match thing:
+                    case OverriddenInstructionPlaceHolder():
+                        # TODO: Is this helpful?
+                        self.write_overridden_instr_place_holder(thing)
```

Comment:
I think it's not (it only matters for generated_cases.c.h IMO), just use `pass`.